Modules/Devices/src/Devices.Application/Infrastructure/PushNotifications/DeletionProcess/DeletionProcessGracePeriodPushNotification.cs

```diff
@@ -1,4 +1,4 @@
 namespace Backbone.Modules.Devices.Application.Infrastructure.PushNotifications.DeletionProcess;
 
 [NotificationText(Title = "Your Identity will be deleted", Body = "Your Identity will be deleted in a few days. You can still cancel up to this point.")]
-public record DeletionProcessGracePeriodNotification(int DaysUntilDeletion);
+public record DeletionProcessGracePeriodPushNotification(int DaysUntilDeletion);
```

Comment:
Please read the task description again and fix the name accordingly...

Reply:
oops, sry